samples/cpp/tutorial_code/ml/non_linear_svms/non_linear_svms.cpp

```diff
@@ -26,13 +21,16 @@ int main()
 {
     help();
 
+    const int NTRAINING_SAMPLES = 100;         // Number of training samples per class
+    const float FRAC_LINEAR_SEP = 0.9f;        // Fraction of samples which compose the linear separable part
+
     // Data for visual representation
     const int WIDTH = 512, HEIGHT = 512;
     Mat I = Mat::zeros(HEIGHT, WIDTH, CV_8UC3);
 
     //--------------------- 1. Set up training data randomly ---------------------------------------
-    Mat trainData(2*NTRAINING_SAMPLES, 2, CV_32FC1);
-    Mat labels   (2*NTRAINING_SAMPLES, 1, CV_32SC1);
```

Comment:
> CV_32FC1

looks better to me in this case:
- `CV_32F` is just a data depth.
- `CV_32FC1` is complete cv::Mat data type (depth + nchannels)
So they are basically not the same things.

BTW, Currently `C1` types has the same direct mappings to data depths (same value).

Reply:
OK, I see. I will avoid this kind of changes.